### PR TITLE
#996: ux cleanup for timestamps

### PR DESCRIPTION
**Pull request by kkoskelin**
`All times should be displayed as hh:mm am ET (need to add ET to all displayed times) and drop the leading 0 when the time is a single digit...`
Screen Shot 2019-10-03 at 4 08 40 PM

`... EXCEPT the time on the Add Trial Session form and Trial Session Detail screen`
Screen Shot 2019-10-03 at 4 07 40 PM

### CODE REVIEW

File: shared/src/business/utilities/DateHandler.js [hunk 1]
```diff
@@ -2,9 +2,11 @@ const moment = require('moment-timezone');
 
 const dateFormats = {
   DATE_TIME: 'MM/DD/YY hh:mm a',
+  DATE_TIME_TZ: 'MM/DD/YY h:mm a [ET]',
   MMDDYY: 'MM/DD/YY',
   MMDDYYYY: 'MM/DD/YYYY',
-  TIME: 'hh:mm a',
+  TIME: 'h:mm a [ET]',
+  TIME_TZ: 'hh:mm a',
```
Comment (rachaelparris) [inline shared/src/business/utilities/DateHandler.js]: Could you make this one `TIME_NO_TZ` or something? It's a little confusing that this one does not have the `ET` portion but `DATE_TIME_TZ` does.

**Review by wbyoko — APPROVED** [review]

File: shared/src/business/utilities/DateHandler.js (hunk 1 above)
Comment (kkoskelin) [reply shared/src/business/utilities/DateHandler.js]: Excellent suggestion.  Thanks!

**Review by rachaelparris — APPROVED** [review]

**Comment by kkoskelin** [comment]
moving back to "In Progress" as some of the outlined test scenarios are still not panning out as I'd hoped.

File: shared/src/business/test/createTestApplicationContext.js [hunk 2]
```diff
@@ -139,6 +140,9 @@ const createTestApplicationContext = ({ user } = {}) => {
     getUniqueId: () => {
       return uuidv4();
     },
+    getUtilities: () => {
+      return { ...DateHandler };
```
Comment (sutt0n) [inline shared/src/business/test/createTestApplicationContext.js:144]: why not just `return DateHandler`?

File: web-client/integration-tests/journey/petitionsClerkViewsUnreadMessage.js [hunk 3]
```diff
@@ -4,6 +4,7 @@ import { withAppContextDecorator } from '../../src/withAppContext';
 
 const formattedWorkQueue = withAppContextDecorator(formattedWorkQueueComputed);
 
+// THIS is the one that's busted
```
Comment (sutt0n) [inline web-client/integration-tests/journey/petitionsClerkViewsUnreadMessage.js]: is this still a relevant comment?

Comment (matthopson) [inline web-client/integration-tests/journey/petitionsClerkViewsUnreadMessage.js]: Do you still need this?

**Review by sutt0n — APPROVED** [review]

File: web-client/src/presenter/computeds/formattedWorkQueue.js [hunk 4]
```diff
@@ -67,11 +63,17 @@ export const formatWorkItem = (
     message.createdAtTimeFormatted = applicationContext
```
Comment (rachaelparris) [inline web-client/src/presenter/computeds/formattedWorkQueue.js]: Do we still need these times without the timezone?

File: web-client/integration-tests/journey/petitionsClerkViewsUnreadMessage.js (hunk 3 above)
Comment (kkoskelin) [reply web-client/integration-tests/journey/petitionsClerkViewsUnreadMessage.js]: aha, thank you!

**Review by codyseibert — APPROVED** [review]

File: web-client/integration-tests/journey/petitionsClerkViewsUnreadMessage.js (hunk 3 above)
Comment (kkoskelin) [reply web-client/integration-tests/journey/petitionsClerkViewsUnreadMessage.js]: pushed a fix

File: web-client/src/presenter/computeds/formattedWorkQueue.js (hunk 4 above)
Comment (kkoskelin) [reply web-client/src/presenter/computeds/formattedWorkQueue.js]: pushed a fix

File: web-client/integration-tests/journey/petitionsClerkViewsUnreadMessage.js (hunk 3 above)
Comment (kkoskelin) [reply web-client/integration-tests/journey/petitionsClerkViewsUnreadMessage.js]: nope, fixed.

File: shared/src/business/test/createTestApplicationContext.js (hunk 2 above)
Comment (kkoskelin) [reply shared/src/business/test/createTestApplicationContext.js:144]: My thinking was that this was possibly a clearer format for the next person who reads this class, demonstrating I"m importing all the functions from DateHandler but folks could follow suit with similar syntax if importing something else.  Are we okay with that, or would you prefer the briefer syntax?

**Review by rachaelparris — APPROVED** [review]

**Review by matthopson — APPROVED** [review]

**Pull request merged**